charts/shoot-cloud-config/charts/downloader/templates/cloud-config/cloud-init.sh

```diff
@@ -0,0 +1,46 @@
+{{- define "shoot-cloud-config.cloud-init-alicloud" -}}
+#!/bin/bash
+
+#Disable locksmithd
+systemctl disable locksmithd
+systemctl stop locksmithd
+
+#Fix mis-configuration of dockerd
+mkdir -p /etc/docker
+echo '{ "storage-driver": "devicemapper" }' > /etc/docker/daemon.json
+sed -i '/Environment=DOCKER_SELINUX=--selinux-enabled=true/s/^/#/g' /run/systemd/system/docker.service
+
+
+#Cloud Config Downloader
+
+#ServiceUnit
+cat <<EOF > "/etc/systemd/system/cloud-config-downloader.service"
+[Unit]
```

Comment:
This part looks similar to what is used for other clouds. Can you please put it in a variable (in a `_helpers.tpl` file or sth) so that we can reuse it?